recipes/python-pdal/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "python-pdal" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "60d41da79441d0aa9c44ec6ebb7497ffc0090d761259d42d4eeb05c7146bb432" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/PDAL/python/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py!=36]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy 1.11.*
+    - packaging
+    - pdal 1.7.*
```

Comment:
Since you're building c-extensions, could you add `toolchain` to the `build` requirements?

Reply:
I've just pushed a version that does the `pip install`. Does adding `toolchain` still buy us anything in that case? Things seem to work just fine without, but maybe there is an underlying reason why it is still needed.

Reply:
@chambbj Yes, it's still wanted to set various compiler flags, for example targeting the correct MacOS SDK version.